CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# How to contribute
+
+If you want to contribute to Ananas, first of all: thank you for your interest!
+
+We use a fairly straightforward development flow:
+- To work on a new feature, fork the repository and create a new feature branch.
+	- If your feature is purely layout/design-related, prefix the branch name with `layout`.
+	For example: `layout-settings-view`.
```

Comment:
You could add something like:

If your branch fixes an issue, please prefix it with the number of the issue.
For example: `34-emoji-keyboard`